components/UpdatedDate/UpdatedDate.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReactNode } from "react"
```

Comment:
```suggestion

```

Reply:
Please @EmmaDawsonDev, can you elaborate on this..